PNA.mdk

```diff
@@ -1057,7 +1056,7 @@ existing slides.
 
 ## Is it inefficient to have the `MainParser` redo work? {#appendix-reparsing}
 
-If the only changes made by the inline extern in the network-to-host
+If the only changes made by the decryption fixed function block in the network-to-host
```

Comment:
In the architecture picture and description what was previously called "inline extern" is now called "inline accelerators". 
This said, the text also states that "inline accelerators are fixed-function blocks", so even though we use different terms for the same element, the terminology seems coherent to me. But I just wanted to bring this to your attention to make sure you are aware of and fine with it.

Reply:
Good point.  It seems worth having a short glossary of terms somewhere, along with preferred terms to use in the spec for consistency's sake.  I will merge this one as is, and create a separate issue about the glossary and whether to use "inline accelerator" in most places in the PNA spec.